pkg/assembler/graphql/model/nodes.go

```diff
@@ -103,4 +112,5 @@ type PkgSpec struct {
 	Name       *string                  `json:"name"`
 	Version    *string                  `json:"version"`
 	Qualifiers []*PackageQualifierInput `json:"qualifiers"`
```

Comment:
just occurred to me that if we want to look for "no qualifiers" we would not be able to differentiate from all qualifiers vs no qualifiers.

Reply:
Right. I have a TODO to think about this but likely it needs a different PR.